sagemaker-pyspark-sdk/requirements.txt

```diff
@@ -1,3 +1,3 @@
 pyspark==2.3.2
 numpy
```

Comment:
Hope this works. Most cases when I see numpy it's bounded with versions.

Reply:
yes. this is pre-existing. i didn't want to make unnecessary extra changes as part of the ci change